harmony/to_clr_convertion/commands.py

```
@@ -12,6 +16,17 @@ def txt2clr(
     suffix: str = core.CommonArguments.suffix,
 ):
     """Command to convert a text file into a ".clr" file"""
-    FromTxtCommandUtils(CLRWriting(), suffix, generate_names).convert_from_txt_file(
-        file_path
-    )
+    file_path_clr = FromTxtCommandUtils(
+        CLRWriting(), suffix, generate_names
+    ).convert_from_txt_file(file_path)
+    _copy_file_macos(file_path_clr)
```

This line is breaking the functionality of the command. This command is only for converting "Harmony" files into `.clr` files. "Installing" `.clr` should be placed in a new command. The new command should also raise an Exception in case the file is not a `.clr` file (doesn't have the `04 0b 73 74 72 65 61 6d 74 79 70 65 64` signature byte string at the beginning of the file) or the current OS is not MacOS, since this functionality is exclusive to Apple operating systems

---

In respect to https://github.com/verocloud/color-sorting-cli/issues/28#issuecomment-1480297615, instead of creating a new command we will add a new flag option (which value will be `False` by default). The option must have a OS checking which will raise an error in case it is used in a OS different from MacOS.

---

Hello xlurio, Do we need created the new flag? I think we can checking os in command logic, if it is macos then i will copy file to Colors library and if not, We run command normally.